### PR TITLE
[MIG] sale_ux: Migration to 17.0

### PR DESCRIPTION
**Pull request by matiasperalta1**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/sale/pull/743).

File: sale_ux/views/res_config_settings_views.xml [hunk 1]
```diff
@@ -1,118 +1,72 @@
...
+                        <field name="group_allow_any_user_as_salesman"/>
+                        <label for="group_allow_any_user_as_salesman"/>
                     </div>
-                    <div class="o_setting_right_pane">
-                        <label for="show_product_image_on_report" string="Show Product Image On Report"/>
-                        <div class="text-muted">
-                            Show Product Image On Report
-                        </div>
+                    <div class="text-muted">
+                        By default odoo only allows to choose users wich have sales pemission in sales orders and in the sales teams only internal users. This option make able to choose, in both places, users of any type (internal or portal)
```
Comment (jok-adhoc) [inline sale_ux/views/res_config_settings_views.xml]: By default, Odoo only allows to choose users who have sales permission in sales orders and only internal users in the sales' teams . This option enables to choose, in both places, users of any type (internal or portal)

File: sale_ux/views/sale_order_views.xml [hunk 2]
```diff
@@ -18,33 +18,27 @@
...
             </xpath>
-
             <group name="note_group" position="after">
                 <newline/>
                 <field name="internal_notes" nolabel="1" placeholder="Internal Notes..." groups="base.group_user"/>
             </group>
 
             <!-- add buttons to send and print for state done too -->
-            <xpath expr="//button[@name='action_quotation_send'][4]" position="attributes">
+            <!-- <xpath expr="//button[@name='action_quotation_send'][4]" position="attributes">
```
Comment (jok-adhoc) [inline sale_ux/views/sale_order_views.xml]: esto no lo borras por algo?

**Comment by jok-adhoc** [comment]
@roboadhoc r+

**Pull request closed without merging**